src/collections/_documentation/guides/manage-event-stream/index.md

```diff
@@ -183,10 +183,10 @@ A good way to set a project rate limit is by figuring out the expected event vol
 - Open the project DSN key configuration under `[Project Settings] > Client Keys > [Configure]`
 - Take a look at the `KEY USAGE IN THE LAST 30 DAYS` graph. Max daily rate in the last month is < 326K
 - Based on that, we can define a ceiling **daily** max value of ~330K, which is ~13,750 events an **hour**.
-- Notice that you can set a daily, hourly, or minute-based rate limit. We'd recommend using an hourly rate to avoid situations where a random event spike might exhaust your daily set quota and leave you blind for the rest of the day.
-- You can always go back, check the graph to see the number of events dropped due to rate limiting and revisit your settings.
+- Notice that you can set a daily, hourly, or minute-based rate limit. We'd recommend using an per minute rate limit to avoid situations where a random event spike might exhaust your daily set quota and leave you blind for an extended period of time.
```

Comment:
```suggestion
- Notice that you can set a daily, hourly, or minute-based rate limit. We'd recommend using a per minute rate limit to avoid situations where a random event spike might exhaust your daily set quota and leave you blind for an extended period of time.
```